packages/design-system/src/components/Table/Table.tsx

```diff
@@ -126,85 +185,42 @@ export class Table extends React.PureComponent {
       className
     );
 
-    // Makes table container focusable and displays the scrollable notice when table width exceeds viewport
-    // by setting `tabIndex = 0` attribute.
-    // This provides context for screen readers to the table's <caption> via aria-labelleby
+    /**
+     * Makes table container focusable and displays the scrollable notice when table width exceeds viewport
+     * by setting `tabIndex = 0` attribute.
+     * This provides context for screen readers to the table's <caption> via aria-labelleby
+     */
     const attributeScrollable = scrollable && {
       className: 'ds-c-table__wrapper',
       role: 'region',
       'aria-labelledby': this.captionID,
-      'aria-live': 'polite',
-      'aria-relevant': 'additions',
-      tabIndex: this.state.scrollActive ? '0' : null,
+      tabIndex: this.state.scrollActive ? 0 : null,
     };
+    const contextValue = { stackable: !!stackable, warningDisabled: !!warningDisabled };
 
-    return (
+    return scrollable ? (
       <div
         ref={(container) => {
           this.container = container;
         }}
+        aria-live="polite"
```

Comment:
Can you annotate the reasons for adding `aria-live` and `aria-relevant` here? Is it because the scroll container is added and scrollbars forced to show? In my component that wraps the DS `Table`, I add aria-live for `stackable`, because it rearranges row headers.